src/renderers/shared/reconciler/ReactCompositeComponent.js

```diff
@@ -498,6 +498,7 @@ var ReactCompositeComponentMixin = {
     var inst = this._instance;
     var childContext = inst.getChildContext && inst.getChildContext();
     if (childContext) {
+      ReactCurrentOwner.processingChildContext = true;
```

Comment:
Cool , Thanks!  

So I notice I can't extend the internalInstance object to add the flag. What is a good object to add the flag to?